<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firmware</class>
 <widget class="QDialog" name="firmware">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware upload</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../onewire.qrc">
    <normaloff>:/images/images/gear_refresh.png</normaloff>:/images/images/gear_refresh.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonGo">
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="6">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonForce">
     <property name="text">
      <string>Force</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonStop">
     <property name="text">
      <string>Abort</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxLog">
     <property name="text">
      <string>Log</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="labelTrafficPix">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../onewire.qrc">:/images/images/ballred.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="labelTrafficTxt">
     <property name="text">
      <string>Standby</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../onewire.qrc"/>
 </resources>
 <connections/>
</ui>
